Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Microsoft 365/Bases/PaletteMicrosoft365Base.cs

```diff
@@ -4900,41 +4900,90 @@ public virtual string BaseFontName
         protected override void DefineFonts()
         {
             // Release existing resources
-            _header1ShortFont?.Dispose();
-
-            _header2ShortFont?.Dispose();
+            if (_header1ShortFont != null)
```

Comment:
Why has this been turned into the long format. That is what the `?.` does for the code.

Reply:
Moved to base